southbound/impl/src/main/java/org/opendaylight/ansible/southbound/AnsibleEventList.java

```diff
@@ -0,0 +1,55 @@
+package org.opendaylight.ansible.southbound;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
```

Comment:
this should fail the checkstyle? Did you do a mvn clean install - without -Pq before pushing?

Reply:
i'll check and fix it